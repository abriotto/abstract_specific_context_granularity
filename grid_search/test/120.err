  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  2%|###7                                                                                                                                                                                        | 2/99 [00:00<00:06, 14.00it/s]  4%|#######5                                                                                                                                                                                    | 4/99 [00:00<00:07, 12.05it/s]  7%|#############2                                                                                                                                                                              | 7/99 [00:00<00:06, 13.34it/s]  9%|#################                                                                                                                                                                           | 9/99 [00:00<00:08, 10.55it/s] 11%|####################7                                                                                                                                                                      | 11/99 [00:00<00:07, 12.25it/s] 13%|########################5                                                                                                                                                                  | 13/99 [00:01<00:08, 10.58it/s] 15%|############################3                                                                                                                                                              | 15/99 [00:01<00:09,  9.15it/s] 17%|################################1                                                                                                                                                          | 17/99 [00:01<00:07, 10.57it/s] 19%|###################################8                                                                                                                                                       | 19/99 [00:01<00:07, 10.85it/s] 21%|#######################################6                                                                                                                                                   | 21/99 [00:02<00:08,  8.76it/s] 23%|###########################################4                                                                                                                                               | 23/99 [00:02<00:08,  8.66it/s] 25%|###############################################2                                                                                                                                           | 25/99 [00:02<00:08,  8.65it/s] 26%|#################################################1                                                                                                                                         | 26/99 [00:02<00:08,  8.13it/s] 27%|##################################################9                                                                                                                                        | 27/99 [00:02<00:09,  7.47it/s] 29%|######################################################7                                                                                                                                    | 29/99 [00:03<00:08,  7.91it/s] 30%|########################################################6                                                                                                                                  | 30/99 [00:03<00:09,  7.53it/s] 31%|##########################################################5                                                                                                                                | 31/99 [00:03<00:09,  7.15it/s] 32%|############################################################4                                                                                                                              | 32/99 [00:03<00:09,  6.78it/s] 34%|################################################################2                                                                                                                          | 34/99 [00:03<00:07,  8.53it/s] 35%|##################################################################1                                                                                                                        | 35/99 [00:03<00:08,  7.82it/s] 36%|####################################################################                                                                                                                       | 36/99 [00:04<00:08,  7.36it/s] 38%|#######################################################################7                                                                                                                   | 38/99 [00:04<00:07,  8.20it/s] 39%|#########################################################################6                                                                                                                 | 39/99 [00:04<00:07,  7.83it/s] 40%|###########################################################################5                                                                                                               | 40/99 [00:04<00:08,  7.34it/s] 42%|###############################################################################3                                                                                                           | 42/99 [00:04<00:06,  8.89it/s] 44%|###################################################################################1                                                                                                       | 44/99 [00:04<00:06,  8.61it/s] 46%|######################################################################################8                                                                                                    | 46/99 [00:05<00:05,  9.88it/s] 48%|##########################################################################################6                                                                                                | 48/99 [00:05<00:05,  9.15it/s] 49%|############################################################################################5                                                                                              | 49/99 [00:05<00:05,  8.46it/s] 51%|##############################################################################################4                                                                                            | 50/99 [00:05<00:06,  7.50it/s] 52%|################################################################################################3                                                                                          | 51/99 [00:05<00:06,  7.20it/s] 53%|##################################################################################################2                                                                                        | 52/99 [00:06<00:06,  6.82it/s] 55%|#####################################################################################################9                                                                                     | 54/99 [00:06<00:05,  8.01it/s] 56%|#######################################################################################################8                                                                                   | 55/99 [00:06<00:05,  7.36it/s] 58%|###########################################################################################################6                                                                               | 57/99 [00:06<00:04,  9.73it/s] 60%|###############################################################################################################4                                                                           | 59/99 [00:06<00:04,  9.69it/s] 62%|###################################################################################################################2                                                                       | 61/99 [00:06<00:03,  9.98it/s] 64%|#######################################################################################################################                                                                    | 63/99 [00:07<00:04,  8.10it/s] 66%|##########################################################################################################################7                                                                | 65/99 [00:07<00:04,  8.14it/s] 68%|##############################################################################################################################5                                                            | 67/99 [00:07<00:03,  9.29it/s] 70%|##################################################################################################################################3                                                        | 69/99 [00:07<00:03,  8.83it/s] 71%|####################################################################################################################################2                                                      | 70/99 [00:08<00:03,  8.20it/s] 74%|#########################################################################################################################################8                                                 | 73/99 [00:08<00:02, 11.06it/s] 76%|#############################################################################################################################################6                                             | 75/99 [00:08<00:02, 10.16it/s] 78%|#################################################################################################################################################4                                         | 77/99 [00:08<00:02,  8.51it/s] 79%|###################################################################################################################################################3                                       | 78/99 [00:08<00:02,  8.03it/s] 81%|#######################################################################################################################################################1                                   | 80/99 [00:09<00:02,  8.14it/s] 83%|##########################################################################################################################################################8                                | 82/99 [00:09<00:02,  7.95it/s] 85%|##############################################################################################################################################################6                            | 84/99 [00:09<00:01,  9.21it/s] 87%|##################################################################################################################################################################4                        | 86/99 [00:09<00:01, 10.10it/s] 89%|######################################################################################################################################################################2                    | 88/99 [00:09<00:01,  9.35it/s] 90%|########################################################################################################################################################################1                  | 89/99 [00:10<00:01,  8.59it/s] 92%|###########################################################################################################################################################################8               | 91/99 [00:10<00:00,  8.37it/s] 93%|#############################################################################################################################################################################7             | 92/99 [00:10<00:00,  7.73it/s] 94%|###############################################################################################################################################################################6           | 93/99 [00:10<00:00,  7.39it/s] 96%|###################################################################################################################################################################################4       | 95/99 [00:10<00:00,  8.77it/s] 97%|#####################################################################################################################################################################################3     | 96/99 [00:11<00:00,  7.96it/s] 98%|#######################################################################################################################################################################################2   | 97/99 [00:11<00:00,  7.50it/s] 99%|#########################################################################################################################################################################################1 | 98/99 [00:11<00:00,  7.16it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  6.81it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  8.60it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s]  4%|#######5                                                                                                                                                                                    | 1/25 [00:00<00:03,  6.38it/s] 12%|######################5                                                                                                                                                                     | 3/25 [00:00<00:02,  9.55it/s] 20%|#####################################6                                                                                                                                                      | 5/25 [00:00<00:02,  9.01it/s] 24%|#############################################1                                                                                                                                              | 6/25 [00:00<00:02,  8.10it/s] 28%|####################################################6                                                                                                                                       | 7/25 [00:00<00:02,  7.42it/s] 36%|###################################################################6                                                                                                                        | 9/25 [00:01<00:02,  7.58it/s] 44%|##################################################################################2                                                                                                        | 11/25 [00:01<00:01,  7.80it/s] 48%|#########################################################################################7                                                                                                 | 12/25 [00:01<00:01,  7.38it/s] 60%|################################################################################################################2                                                                          | 15/25 [00:01<00:00, 10.20it/s] 68%|###############################################################################################################################1                                                           | 17/25 [00:01<00:00,  9.99it/s] 76%|##############################################################################################################################################1                                            | 19/25 [00:02<00:00, 10.86it/s] 84%|#############################################################################################################################################################                              | 21/25 [00:02<00:00,  9.79it/s] 92%|############################################################################################################################################################################               | 23/25 [00:02<00:00, 10.50it/s]100%|###########################################################################################################################################################################################| 25/25 [00:02<00:00, 11.16it/s]100%|###########################################################################################################################################################################################| 25/25 [00:02<00:00,  9.47it/s]
