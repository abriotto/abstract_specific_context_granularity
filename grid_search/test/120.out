# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.97", "--temperature=1"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6960024833679199, "acc": 0.519240140914917, "length": 3.5717074871063232, "mode": "train", "epoch": 1}
{"loss": 0.6915280222892761, "acc": 0.5243548154830933, "length": 1.0032894611358643, "mode": "test", "epoch": 1}
{"loss": 0.6910437345504761, "acc": 0.5258703827857971, "length": 3.499931573867798, "mode": "train", "epoch": 2}
{"loss": 0.6840111613273621, "acc": 0.5588668584823608, "length": 1.1644736528396606, "mode": "test", "epoch": 2}
{"loss": 0.6852328181266785, "acc": 0.554101824760437, "length": 3.3900933265686035, "mode": "train", "epoch": 3}
{"loss": 0.6649503707885742, "acc": 0.6302511096000671, "length": 3.205592155456543, "mode": "test", "epoch": 3}
{"loss": 0.6609371900558472, "acc": 0.6263715028762817, "length": 3.5986270904541016, "mode": "train", "epoch": 4}
{"loss": 0.6358150839805603, "acc": 0.6689966320991516, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.61210697889328, "acc": 0.6657594442367554, "length": 3.921252965927124, "mode": "train", "epoch": 5}
{"loss": 0.6139628291130066, "acc": 0.6559211015701294, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.5856359601020813, "acc": 0.6902856826782227, "length": 3.959249258041382, "mode": "train", "epoch": 6}
{"loss": 0.6100323796272278, "acc": 0.6798520684242249, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.5626381635665894, "acc": 0.7254610061645508, "length": 3.9506654739379883, "mode": "train", "epoch": 7}
{"loss": 0.5705758333206177, "acc": 0.7224507331848145, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.5289789438247681, "acc": 0.74789959192276, "length": 3.9679059982299805, "mode": "train", "epoch": 8}
{"loss": 0.5170473456382751, "acc": 0.7541118860244751, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.48960551619529724, "acc": 0.7671356797218323, "length": 3.9911460876464844, "mode": "train", "epoch": 9}
{"loss": 0.4889093339443207, "acc": 0.774917721748352, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.4612540304660797, "acc": 0.7879132032394409, "length": 3.9940857887268066, "mode": "train", "epoch": 10}
{"loss": 0.4749790132045746, "acc": 0.7816612124443054, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.4401548206806183, "acc": 0.7984188199043274, "length": 3.9935245513916016, "mode": "train", "epoch": 11}
{"loss": 0.4617193639278412, "acc": 0.788404643535614, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.4226951599121094, "acc": 0.8080371022224426, "length": 3.995555877685547, "mode": "train", "epoch": 12}
{"loss": 0.4470451772212982, "acc": 0.7990131378173828, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.4060436487197876, "acc": 0.820273756980896, "length": 3.992436408996582, "mode": "train", "epoch": 13}
{"loss": 0.426603764295578, "acc": 0.8102795481681824, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.39318975806236267, "acc": 0.8308128118515015, "length": 3.997687816619873, "mode": "train", "epoch": 14}
{"loss": 0.4177190959453583, "acc": 0.8274669051170349, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.38268980383872986, "acc": 0.8398130536079407, "length": 3.998488187789917, "mode": "train", "epoch": 15}
{"loss": 0.40051040053367615, "acc": 0.830592155456543, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.37597301602363586, "acc": 0.8433482646942139, "length": 3.9979910850524902, "mode": "train", "epoch": 16}
{"loss": 0.394542932510376, "acc": 0.8356908559799194, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.36062008142471313, "acc": 0.8478354811668396, "length": 3.9967124462127686, "mode": "train", "epoch": 17}
{"loss": 0.3886162340641022, "acc": 0.8397204875946045, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.3517005145549774, "acc": 0.8544889092445374, "length": 3.9972901344299316, "mode": "train", "epoch": 18}
{"loss": 0.3688454329967499, "acc": 0.8491774201393127, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.3418356478214264, "acc": 0.8575439453125, "length": 3.995042324066162, "mode": "train", "epoch": 19}
{"loss": 0.3703274130821228, "acc": 0.8509867787361145, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.34152281284332275, "acc": 0.8588579297065735, "length": 3.9980556964874268, "mode": "train", "epoch": 20}
{"loss": 0.3542027175426483, "acc": 0.853618323802948, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.33008241653442383, "acc": 0.8637832403182983, "length": 3.9974067211151123, "mode": "train", "epoch": 21}
{"loss": 0.3536740243434906, "acc": 0.8553454875946045, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.32889968156814575, "acc": 0.8678349852561951, "length": 3.997457981109619, "mode": "train", "epoch": 22}
{"loss": 0.3369257152080536, "acc": 0.8692435026168823, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.31418710947036743, "acc": 0.8734728097915649, "length": 3.9990427494049072, "mode": "train", "epoch": 23}
{"loss": 0.33064156770706177, "acc": 0.872450590133667, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.30649039149284363, "acc": 0.8764137625694275, "length": 3.9971771240234375, "mode": "train", "epoch": 24}
{"loss": 0.32286596298217773, "acc": 0.8742598295211792, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.3037913739681244, "acc": 0.8804651498794556, "length": 3.9975028038024902, "mode": "train", "epoch": 25}
{"loss": 0.32241541147232056, "acc": 0.8784540295600891, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.3019746243953705, "acc": 0.877240777015686, "length": 3.9984655380249023, "mode": "train", "epoch": 26}
{"loss": 0.3193005323410034, "acc": 0.877713680267334, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.29571717977523804, "acc": 0.8808065056800842, "length": 3.9973764419555664, "mode": "train", "epoch": 27}
{"loss": 0.3133337199687958, "acc": 0.8808385729789734, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.2860803008079529, "acc": 0.8865953683853149, "length": 3.9996936321258545, "mode": "train", "epoch": 28}
{"loss": 0.3108644187450409, "acc": 0.8816609382629395, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.28861528635025024, "acc": 0.8874332308769226, "length": 3.9990780353546143, "mode": "train", "epoch": 29}
{"loss": 0.30480170249938965, "acc": 0.8873355388641357, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.2815150320529938, "acc": 0.8896287679672241, "length": 3.999229907989502, "mode": "train", "epoch": 30}
{"loss": 0.29993751645088196, "acc": 0.8853616118431091, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.2773478031158447, "acc": 0.8917868733406067, "length": 3.999682903289795, "mode": "train", "epoch": 31}
{"loss": 0.31308913230895996, "acc": 0.882154643535614, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.2722821831703186, "acc": 0.8947620987892151, "length": 3.998258113861084, "mode": "train", "epoch": 32}
{"loss": 0.297779381275177, "acc": 0.8875000476837158, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.26697680354118347, "acc": 0.897026777267456, "length": 3.998964786529541, "mode": "train", "epoch": 33}
{"loss": 0.287017822265625, "acc": 0.8930100798606873, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.26020345091819763, "acc": 0.8996745944023132, "length": 3.9986579418182373, "mode": "train", "epoch": 34}
{"loss": 0.2947659194469452, "acc": 0.8888980746269226, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.25159963965415955, "acc": 0.9042885899543762, "length": 3.9994616508483887, "mode": "train", "epoch": 35}
{"loss": 0.28249502182006836, "acc": 0.8951479196548462, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.24998915195465088, "acc": 0.9057023525238037, "length": 3.9983019828796387, "mode": "train", "epoch": 36}
{"loss": 0.2747702896595001, "acc": 0.9005757570266724, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.2512874901294708, "acc": 0.9033373594284058, "length": 3.998504877090454, "mode": "train", "epoch": 37}
{"loss": 0.27027440071105957, "acc": 0.8971217274665833, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.24397847056388855, "acc": 0.9071848392486572, "length": 3.999018669128418, "mode": "train", "epoch": 38}
{"loss": 0.2606644630432129, "acc": 0.9023026823997498, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.23853187263011932, "acc": 0.9081173539161682, "length": 3.9989511966705322, "mode": "train", "epoch": 39}
{"loss": 0.2571864128112793, "acc": 0.9007400870323181, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.2344771772623062, "acc": 0.9099466800689697, "length": 3.999610424041748, "mode": "train", "epoch": 40}
{"loss": 0.26038581132888794, "acc": 0.9023848176002502, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.23590794205665588, "acc": 0.9110456705093384, "length": 3.9990808963775635, "mode": "train", "epoch": 41}
{"loss": 0.26336607336997986, "acc": 0.8974506258964539, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.23222680389881134, "acc": 0.9139814376831055, "length": 3.998103141784668, "mode": "train", "epoch": 42}
{"loss": 0.25531134009361267, "acc": 0.9063323140144348, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.2263335883617401, "acc": 0.9156948924064636, "length": 3.9980404376983643, "mode": "train", "epoch": 43}
{"loss": 0.25114402174949646, "acc": 0.9042763710021973, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.22342944145202637, "acc": 0.9164396524429321, "length": 3.9990804195404053, "mode": "train", "epoch": 44}
{"loss": 0.24790529906749725, "acc": 0.9116777181625366, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.2200961709022522, "acc": 0.9181909561157227, "length": 3.9987125396728516, "mode": "train", "epoch": 45}
{"loss": 0.24879251420497894, "acc": 0.9095394611358643, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.22301708161830902, "acc": 0.918927788734436, "length": 3.999587059020996, "mode": "train", "epoch": 46}
{"loss": 0.24862082302570343, "acc": 0.9078948497772217, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.22166086733341217, "acc": 0.9148549437522888, "length": 3.9992597103118896, "mode": "train", "epoch": 47}
{"loss": 0.2612588107585907, "acc": 0.901069164276123, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.2170020490884781, "acc": 0.9169995784759521, "length": 3.999617099761963, "mode": "train", "epoch": 48}
{"loss": 0.2562255859375, "acc": 0.9075658917427063, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.21378843486309052, "acc": 0.9205195903778076, "length": 3.9997572898864746, "mode": "train", "epoch": 49}
{"loss": 0.24102595448493958, "acc": 0.9164472818374634, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.210686594247818, "acc": 0.9239086508750916, "length": 3.9992105960845947, "mode": "train", "epoch": 50}
{"loss": 0.24316084384918213, "acc": 0.9121711254119873, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.2124098241329193, "acc": 0.9230828285217285, "length": 3.9984679222106934, "mode": "train", "epoch": 51}
{"loss": 0.24400444328784943, "acc": 0.9125823974609375, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.21057870984077454, "acc": 0.9242134690284729, "length": 3.999619483947754, "mode": "train", "epoch": 52}
{"loss": 0.2374543696641922, "acc": 0.9143091440200806, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.2034568041563034, "acc": 0.925025224685669, "length": 3.9999446868896484, "mode": "train", "epoch": 53}
{"loss": 0.24470344185829163, "acc": 0.9146382212638855, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.20119866728782654, "acc": 0.9259229898452759, "length": 3.99983286857605, "mode": "train", "epoch": 54}
{"loss": 0.2540416717529297, "acc": 0.9099505543708801, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.20443302392959595, "acc": 0.9236767292022705, "length": 3.9999032020568848, "mode": "train", "epoch": 55}
{"loss": 0.23970739543437958, "acc": 0.9140625, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.20418435335159302, "acc": 0.9257170557975769, "length": 3.99900484085083, "mode": "train", "epoch": 56}
{"loss": 0.23178119957447052, "acc": 0.9195724129676819, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.206117182970047, "acc": 0.9247512817382812, "length": 3.9999608993530273, "mode": "train", "epoch": 57}
{"loss": 0.233070969581604, "acc": 0.9214637875556946, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.19746033847332, "acc": 0.9259476065635681, "length": 3.999502658843994, "mode": "train", "epoch": 58}
{"loss": 0.2312777042388916, "acc": 0.9185855388641357, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.19408917427062988, "acc": 0.9314974546432495, "length": 3.999807357788086, "mode": "train", "epoch": 59}
{"loss": 0.2326311618089676, "acc": 0.9141448140144348, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.19205182790756226, "acc": 0.9281697273254395, "length": 3.9997804164886475, "mode": "train", "epoch": 60}
{"loss": 0.2262250930070877, "acc": 0.921875, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.196952685713768, "acc": 0.9270007610321045, "length": 3.99995756149292, "mode": "train", "epoch": 61}
{"loss": 0.22990091145038605, "acc": 0.919819176197052, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.19557221233844757, "acc": 0.9277973771095276, "length": 3.9999639987945557, "mode": "train", "epoch": 62}
{"loss": 0.22460834681987762, "acc": 0.9236017465591431, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.18935173749923706, "acc": 0.9326216578483582, "length": 3.9998438358306885, "mode": "train", "epoch": 63}
{"loss": 0.22354820370674133, "acc": 0.929029643535614, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.1873146891593933, "acc": 0.9314724802970886, "length": 3.999941110610962, "mode": "train", "epoch": 64}
{"loss": 0.22453990578651428, "acc": 0.9229438900947571, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.18840919435024261, "acc": 0.9332211017608643, "length": 3.9996650218963623, "mode": "train", "epoch": 65}
{"loss": 0.2179756909608841, "acc": 0.9195724129676819, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.18302279710769653, "acc": 0.9332315325737, "length": 3.9999771118164062, "mode": "train", "epoch": 66}
{"loss": 0.23104532063007355, "acc": 0.9254933595657349, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.1838882863521576, "acc": 0.9330590963363647, "length": 3.9997897148132324, "mode": "train", "epoch": 67}
{"loss": 0.2215433418750763, "acc": 0.9208880066871643, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.183818981051445, "acc": 0.9347902536392212, "length": 3.999129295349121, "mode": "train", "epoch": 68}
{"loss": 0.22356171905994415, "acc": 0.9236019253730774, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.17949463427066803, "acc": 0.935289204120636, "length": 3.9999635219573975, "mode": "train", "epoch": 69}
{"loss": 0.21496479213237762, "acc": 0.9252467751502991, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.17721135914325714, "acc": 0.9364368319511414, "length": 3.9999008178710938, "mode": "train", "epoch": 70}
{"loss": 0.21961615979671478, "acc": 0.9247533679008484, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.17980416119098663, "acc": 0.9359239935874939, "length": 3.9987409114837646, "mode": "train", "epoch": 71}
{"loss": 0.21776007115840912, "acc": 0.924506425857544, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.1795085221529007, "acc": 0.9359705448150635, "length": 3.9998931884765625, "mode": "train", "epoch": 72}
{"loss": 0.22935552895069122, "acc": 0.9239309430122375, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.1829223483800888, "acc": 0.9348685145378113, "length": 3.9998459815979004, "mode": "train", "epoch": 73}
{"loss": 0.23272617161273956, "acc": 0.9247533679008484, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.18078112602233887, "acc": 0.9356734752655029, "length": 3.999978542327881, "mode": "train", "epoch": 74}
{"loss": 0.2280893325805664, "acc": 0.9246709942817688, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.1803634762763977, "acc": 0.9365705847740173, "length": 3.9997599124908447, "mode": "train", "epoch": 75}
{"loss": 0.22116230428218842, "acc": 0.9222862124443054, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.18255694210529327, "acc": 0.9334325790405273, "length": 3.9987645149230957, "mode": "train", "epoch": 76}
{"loss": 0.21590523421764374, "acc": 0.9255756735801697, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.17555870115756989, "acc": 0.9372538328170776, "length": 3.9998207092285156, "mode": "train", "epoch": 77}
{"loss": 0.21353331208229065, "acc": 0.9227796792984009, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.17285534739494324, "acc": 0.9381913542747498, "length": 3.999877452850342, "mode": "train", "epoch": 78}
{"loss": 0.2092270702123642, "acc": 0.9279604554176331, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.16984957456588745, "acc": 0.9384088516235352, "length": 3.999692440032959, "mode": "train", "epoch": 79}
{"loss": 0.2152515947818756, "acc": 0.9234375357627869, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.16575537621974945, "acc": 0.9391301870346069, "length": 3.9999635219573975, "mode": "train", "epoch": 80}
{"loss": 0.21003501117229462, "acc": 0.9240954518318176, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.16985616087913513, "acc": 0.9413816332817078, "length": 3.9999632835388184, "mode": "train", "epoch": 81}
{"loss": 0.2101982831954956, "acc": 0.9281247854232788, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.17008794844150543, "acc": 0.9400926232337952, "length": 3.9999887943267822, "mode": "train", "epoch": 82}
{"loss": 0.20814958214759827, "acc": 0.9298520088195801, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.16445836424827576, "acc": 0.9396973252296448, "length": 3.999863624572754, "mode": "train", "epoch": 83}
{"loss": 0.20960178971290588, "acc": 0.9276315569877625, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.1675032526254654, "acc": 0.9401735067367554, "length": 3.9999542236328125, "mode": "train", "epoch": 84}
{"loss": 0.20620445907115936, "acc": 0.9288652539253235, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.16275794804096222, "acc": 0.9406955242156982, "length": 3.999981164932251, "mode": "train", "epoch": 85}
{"loss": 0.20475132763385773, "acc": 0.928207278251648, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.16281481087207794, "acc": 0.9410449266433716, "length": 3.999887466430664, "mode": "train", "epoch": 86}
{"loss": 0.1999240517616272, "acc": 0.9310033917427063, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.15822164714336395, "acc": 0.9433168172836304, "length": 3.9997901916503906, "mode": "train", "epoch": 87}
{"loss": 0.21281693875789642, "acc": 0.9261513948440552, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.15794815123081207, "acc": 0.9450304508209229, "length": 3.9999520778656006, "mode": "train", "epoch": 88}
{"loss": 0.20801830291748047, "acc": 0.9267271757125854, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.1622612625360489, "acc": 0.9429168105125427, "length": 3.999962329864502, "mode": "train", "epoch": 89}
{"loss": 0.20471079647541046, "acc": 0.9318257570266724, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.16450564563274384, "acc": 0.9421234130859375, "length": 3.9999845027923584, "mode": "train", "epoch": 90}
{"loss": 0.2040589153766632, "acc": 0.9293583035469055, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.18340055644512177, "acc": 0.9308041334152222, "length": 3.9994633197784424, "mode": "train", "epoch": 91}
{"loss": 0.26437216997146606, "acc": 0.9007400870323181, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.1978350430727005, "acc": 0.9253270030021667, "length": 3.9994101524353027, "mode": "train", "epoch": 92}
{"loss": 0.22704048454761505, "acc": 0.9259867668151855, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.18691633641719818, "acc": 0.9310827255249023, "length": 3.9997920989990234, "mode": "train", "epoch": 93}
{"loss": 0.2305208295583725, "acc": 0.9202302694320679, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.17662368714809418, "acc": 0.9372718334197998, "length": 3.9994490146636963, "mode": "train", "epoch": 94}
{"loss": 0.2219347506761551, "acc": 0.9201479554176331, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.1686033308506012, "acc": 0.9400007128715515, "length": 3.9998955726623535, "mode": "train", "epoch": 95}
{"loss": 0.21208103001117706, "acc": 0.9300163388252258, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.1629626750946045, "acc": 0.9417611956596375, "length": 3.9999265670776367, "mode": "train", "epoch": 96}
{"loss": 0.20679457485675812, "acc": 0.9289473295211792, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.1657755821943283, "acc": 0.9416060447692871, "length": 3.999236822128296, "mode": "train", "epoch": 97}
{"loss": 0.19909891486167908, "acc": 0.9327303171157837, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.1619882583618164, "acc": 0.9448674917221069, "length": 3.9999849796295166, "mode": "train", "epoch": 98}
{"loss": 0.20768573880195618, "acc": 0.9276315569877625, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.15834170579910278, "acc": 0.9454130530357361, "length": 3.9998836517333984, "mode": "train", "epoch": 99}
{"loss": 0.20264171063899994, "acc": 0.9326480031013489, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.15508264303207397, "acc": 0.945909857749939, "length": 3.999979257583618, "mode": "train", "epoch": 100}
{"loss": 0.20046105980873108, "acc": 0.9288651347160339, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.7349123358726501
